mava/utils/make_env.py

```diff
@@ -131,6 +133,41 @@ def make_jaxmarl_env(
     return env, eval_env
 
 
+def make_gigastep_env(
+    env_name: str, config: DictConfig, add_global_state: bool = False
+) -> Tuple[Environment, Environment]:
+    """
+     Create a Gigastep environment.
+
+    Args:
+        env_name (str): The name of the environment to create.
+        config (Dict): The configuration of the environment.
+        add_global_state (bool): Whether to add the global state to the observation. Default False.
+
+    Returns:
+        A tuple of the environments.
+    """
+    kwargs = config.env.kwargs
+
+    scenario_config = config.env.scenario.task_config
+    scenario = ScenarioBuilder.from_config(scenario_config)
+
+    train_env = GigastepWrapper(
+        scenario.make(discrete_actions=True, obs_type="vector"),
```

Comment:
Please add these to the environment config instead and add a comment that we support vector obs and discrete actions for now.